docs/user-interface/controls/shadow.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Shadow"
+description: "Learn how to use the .NET MAUI Shadow class, which paints a shadow around a control."
+ms.date: 10/22/2021
+---
+
+# Shadow
+
+<!-- Sample link, if any, goes here -->
+
+The .NET Multi-platform App UI (.NET MAUI) `Shadow` is a control that paints a shadow around a layout or view. The `VisualElement` class has a `Shadow` bindable property, of type `Shadow`, that enables a shadow to be added to any layout or view.
+
+The `Shadow` class defines the following properties:
+
+- `Radius`, of type `float`, defines the radius of the blur used to generate the shadow. The default value of this property is 10.
+- `Opacity`, of type `float`, indicates the opacity of the shadow. The default value of this property is 1.
+- `Brush`, of type `Brush`, represents the brush used to colorize the shadow.
+- `OffSet`, of type `Point`, specifies the offset for the shadow, which represents the position of the light source that creates the shadow.
+
+These properties are backed by `BindableProperty` objects, which means that they can be targets of data bindings, and styled.
+
+## Create a Shadow
+
+To add a shadow to a control, set the control's `Shadow` property to a `Shadow` object whose properties define its appearance.
+
+The following XAML example shows how to add a shadow to an `Image`:
+
+```xaml
+<Image Source="dotnet_bot.png"
+       WidthRequest="250"
+       HeightRequest="310">
+    <Image.Shadow>
+        <Shadow Brush="Black"
+                Offset="20,20"
+                Radius="40"
+                Opacity="0.8" />
+    </Image.Shadow>
+</Image>
+```
+
+In this example, a black shadow is added to the right and bottom of the image:
+
+:::image type="content" source="media/shadow/image.png" alt-text="Screenshot of an image with a shadow applied.":::
+
+Shadows can also be added to clipped objects, as shown in the following example:
+
+```xaml
+<Image Source=https://aka.ms/campus.jpg
+       Aspect="AspectFill"
+       HeightRequest="220"
+       WidthRequest="220"
+       HorizontalOptions="Center">
+    <Image.Clip>
+        <EllipseGeometry Center="220,250"
+                         RadiusX="220"
+                         RadiusY="220" />
+    </Image.Clip>
+    <Image.Shadow>
+        <Shadow Brush="Black"
+                Offset="10,10"
+                Opacity="0.8" />
+    </Image.Shadow>
+</Image>
+```
+
+In this example, the image is clipped using an `EllipseGeometry`, and a shadow is added to the clipped image:
+
+:::image type="content" source="media/shadow/clipped-image.png" alt-text="Screenshot of a clipped image with a shadow applied.":::
+
+<!-- For more information about clipping an element, see [Clip with a Geometry](). -->
+
+## Create a Shadow gradient
+
+The color of a shadow is defined using a `Brush`. Therefore, gradient shadows can also be added to controls:
```

Comment:
Important detail, the color is defined by a Brush BUT, only **SolidColorBrush** is supported currently.

Reply:
Well, all the brushes are implemented but, for now, only on Android.